controlplane/kubeadm/api/v1beta1/kubeadm_control_plane_types.go

```diff
@@ -176,29 +176,29 @@ type KubeadmControlPlaneStatus struct {
 	// Total number of non-terminated machines targeted by this control plane
 	// that have the desired template spec.
 	// +optional
-	UpdatedReplicas int32 `json:"updatedReplicas"`
+	UpdatedReplicas int32 `json:"updatedReplicas,omitempty"`
```

Comment:
/hold

We have to take a look if changing this breaks some of our code. E.g the topology controller reads KCP with unstructured and then has logic on the field being there or not